src/planner/binder/tableref/bind_table_function.cpp

```diff
@@ -23,276 +23,282 @@
 
 namespace duckdb {
 
-static bool IsTableInTableOutFunction(TableFunctionCatalogEntry &table_function) {
```

Comment:
Why is pretty much everything here marked as changed?
Maybe you need to run `make format-main`

Reply:
This was after I ran make format-fix (I didn't manually change anything in this file within the commit). I now ran make format-main

Reply:
They perform pretty much the same logic, so that's alright, thanks 👍